### PR TITLE
Documentation: warn membership change while migration

### PR DESCRIPTION
**Pull request by gyuho**
Fix https://github.com/coreos/etcd/issues/7429.

### CODE REVIEW

**Comment by xiang90** [comment]
LGTM.

defer to @heyitsanthony

File: Documentation/upgrades/upgrade_3_0.md [hunk 1]
```diff
@@ -124,5 +124,6 @@ $ ETCDCTL_API=3 etcdctl endpoint health
 
 - etcd &lt; v3.1 does not work properly if built with Go &gt; v1.7. See [Issue 6951](https://github.com/coreos/etcd/issues/6951) for additional information.
 - If an error such as `transport: http2Client.notifyError got notified that the client transport was broken unexpected EOF.` shows up in the etcd server logs, be sure etcd is a pre-built release or built with (etcd v3.1+ &amp; go v1.7+) or (etcd &lt;v3.1 &amp; go v1.6.x).
+- Adding a v3 node to v2.3 cluster during upgrades is not supported, and could trigger panics. See [Issue 7249](https://github.com/coreos/etcd/issues/7429) for additional information. Mixed versions of etcd members are only allowed during v3 migration. Finish upgrades before making any membership changes.
```
Comment (heyitsanthony) [inline Documentation/upgrades/upgrade_3_0.md]: s/, and/ and/

**Comment by heyitsanthony** [comment]
lgtm after nit fixup. Thanks!

File: Documentation/upgrades/upgrade_3_0.md [hunk 2]
```diff
@@ -124,5 +124,6 @@ $ ETCDCTL_API=3 etcdctl endpoint health
 
 - etcd &lt; v3.1 does not work properly if built with Go &gt; v1.7. See [Issue 6951](https://github.com/coreos/etcd/issues/6951) for additional information.
 - If an error such as `transport: http2Client.notifyError got notified that the client transport was broken unexpected EOF.` shows up in the etcd server logs, be sure etcd is a pre-built release or built with (etcd v3.1+ &amp; go v1.7+) or (etcd &lt;v3.1 &amp; go v1.6.x).
```
Comment (fanminshi) [inline Documentation/upgrades/upgrade_3_0.md]: for `(etcd v3.1+ & go v1.7+)`, what does `+` means?

Comment (gyuho) [inline Documentation/upgrades/upgrade_3_0.md]: It means `1.7.0, 1.7.1, ...`

Comment (fanminshi) [inline Documentation/upgrades/upgrade_3_0.md]: is it same as 1.7.x?

Comment (fanminshi) [inline Documentation/upgrades/upgrade_3_0.md]: I saw mix usage of `+` and  `v.a.b.x` e.g `go v1.6.x`. I am unsure if they represent same thing.

Comment (gyuho) [inline Documentation/upgrades/upgrade_3_0.md]: Yeah semantically they are the same. I think they all make sense.

Comment (heyitsanthony) [inline Documentation/upgrades/upgrade_3_0.md]: 1.7+ is 1.7 and above (e.g., 1.8). 1.7.x is all versions of 1.7 (e.g., 1.7.1 but not 1.8.0)

**Pull request merged**